pkg/proxy/winkernel/proxier.go

```diff
@@ -1305,3 +1305,13 @@ type endpointServicePair struct {
 	endpoint        string
 	servicePortName proxy.ServicePortName
 }
+
+// Gets the time since the specified start in microseconds.
+func sinceInMicroseconds(start time.Time) float64 {
```

Comment:
Both methods are already present in https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/metrics/metrics.go#L159-L164 any reason to not use them?

Reply:
I didn't know those existed. Changed!